### PR TITLE
LTD-3978-UAT-fix 

### PR DESCRIPTION
**Pull request by seijihg**

### CODE REVIEW

File: api/letter_templates/migrations/0009_refusal_letter_update_fix.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+from django.db import migrations
+from api.picklists.enums import PicklistType
+
+
+def update_refusal_letter_template_paragraphs_fix(apps, schema_editor):
+    PicklistItem = apps.get_model("picklists", "PicklistItem")
+    Team = apps.get_model("teams", "Team")
+
+    admin = Team.objects.get(name="Admin")
+    record = PicklistItem.objects.filter(
```
Comment (currycoder) [inline api/letter_templates/migrations/0009_refusal_letter_update_fix.py:10]: Could I suggest calling this `original_record`?

File: api/letter_templates/migrations/0009_refusal_letter_update_fix.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        text__iexact="Dear,\r\n\r\nHaving carefully considered your application, an export licence has been refused for the goods listed in the attached schedule. If you would like further information about the decision, please contact Jeanette Rosenberg at jeanette.rosenberg@trade.gov.uk, (tel: 07917 751 668).\r\n\r\nYou may appeal against this decision, but you must do so in writing within 28 calendar days of the date of this refusal letter. In doing so you must provide argument or information that was not available to us at the time of refusal, and which could materially affect the decision to refuse.\r\n\r\nAppeal letters should be sent by email to Colin Baker at colin.baker@trade.gov.uk. If you have any questions about the progress of your appeal, please contact Colin (tel: 07391 864 815).\r\n\r\nFor licence applications refused under the Strategic Export Licensing Criteria more information is available at:\r\nhttps://questions-statements.parliament.uk/written-statements/detail/2021-12-08/hcws449.\r\n\r\nYours sincerely,\r\n\r\nExport Control Joint Unit"
+    )
+    if record:
```
Comment (currycoder) [inline api/letter_templates/migrations/0009_refusal_letter_update_fix.py:13]: `if original_record.exists():`

File: api/letter_templates/migrations/0009_refusal_letter_update_fix.py [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+        PicklistItem.objects.filter(name="Refusal letter content").exclude(id=record.first().id).delete()
```
Comment (currycoder) [inline api/letter_templates/migrations/0009_refusal_letter_update_fix.py:14]: A comment here would be helpful; "Delete all refusal letter content records but the original"

File: api/letter_templates/migrations/0009_refusal_letter_update_fix.py (hunk 2 above)
Comment (seijihg) [reply api/letter_templates/migrations/0009_refusal_letter_update_fix.py]: done!

File: api/letter_templates/migrations/tests/test_migration_0009_refusal_letter_update_fix.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            text="Test 2",
+        )
+
+    def test_migration_0009_refusal_letter_update_fix(self):
+        PicklistItem = self.new_state.apps.get_model("picklists", "PicklistItem")
+
+        pick_list_queryset = PicklistItem.objects.filter(name="Refusal letter content")
+
+        assert pick_list_queryset.count() == 1
+        assert (
```
Comment (currycoder) [inline api/letter_templates/migrations/tests/test_migration_0009_refusal_letter_update_fix.py:36]: Can we make sure this is the PicklistItem created first during `prepare`?

Comment (seijihg) [reply api/letter_templates/migrations/tests/test_migration_0009_refusal_letter_update_fix.py:39]: done just added an extra assertion

**Review by currycoder — APPROVED** [review]

**Review by Tllew — APPROVED** [review]
Looks good to me

**Pull request merged**